test/e2e/configure_cni_test.go

```diff
@@ -25,6 +26,10 @@ func TestConfigureCNI(t *testing.T) {
 	t.Run("Configure CNI", testConfigureCNI)
 }
 
+func TestKubeletUninstall(t *testing.T) {
```

Comment:
It is odd that `TestKubeletUninstall` is added to `configure_cni_test.go`. Shouldn't this be a separate test file?